### PR TITLE
Adds Dominian religious clothes.

### PR DESCRIPTION
**Pull request by TheStryker**
Adds clothes from the Moroz Holy Tribunal with lore descriptions done by Schwann. Special thanks to alsoandanswer for helping with some of the sprites.

HAIL DOMINIA

image

I'm willing to tweak the sprites if it's felt that it's necessary.

### CODE REVIEW

**Comment by TheStryker** [comment]
!review

File: code/modules/clothing/factions/dominia.dm [hunk 1]
```diff
@@ -261,3 +261,85 @@
...
+	name = "tribunalist surcoat"
+	desc = "A simple red surcoat commonly worn by Dominian clergy members."
+	desc_fluff = "Spun with rough but hardy fabrics from the Dominian frontier, this surcoat is commonly worn by poorer Tribunal clergy as well as missionaries\
+	seeking protection from the elements. This garment was popularized by the Kael'kah sect and remains respected as a symbol of humility and poverty amongst priests."
+	icon = 'icons/clothing/suits/capes/dominia_surcoat.dmi'
+	icon_state = "dominian_surcoat"
+	item_state = "dominian_surcoat"
+	overlay_state = "dominian_surcoat"
+	icon_override = null
+	contained_sprite = TRUE
```
Comment (alsoandanswer) [inline code/modules/clothing/factions/dominia.dm:274]: If the dominian parent accessory already has contained sprite you dont need to copypaste here

**Review by Geevies — APPROVED** [review]

File: code/modules/clothing/factions/dominia.dm (hunk 1 above)
Comment (NonQueueingMatt) [reply code/modules/clothing/factions/dominia.dm:274]: There is no parent

**Review by NonQueueingMatt — APPROVED** [review]

File: code/modules/clothing/factions/dominia.dm (hunk 1 above)
Comment (TheStryker) [reply code/modules/clothing/factions/dominia.dm]: There is now.

**Comment by TheStryker** [comment]
image

Added a Tribunal amulet to the loadout. Like the locket, it has no sprite when attached to a jumpsuit to represent its small size.

**Pull request merged**